### ISSUE TITLE
Missing encore_entry_link_tags and encore_entry_script_tags in $response->getContent()

### ISSUE THREAD

**Issue by erighetto** [franzwilding/last#1]
I noticed that in SiteGenerator service method generate(array $requests, string $dist_folder) when symfony/webpack-encore-bundle is enabled only for the first handled request the $response->getContent() contains the generated {{ encore_entry_link_tags('app') }} and {{ encore_entry_script_tags('app') }} content.

Es. 
Suppose we have 10 declared route in a custom route  provider, like https://github.com/franzwilding/last#request-providers
For the first request the two above mentioned twig tags are present in $response->getContent();

In the other 9 responses encore entries were not rendered.

It's a router cache issue?

**Comment by franzwilding** [comment]
Thank you! I will have a look

**Comment by erighetto** [comment]
Investigating about this issue I found that the unwanted behaviour begin in https://github.com/symfony/webpack-encore-bundle/blob/master/src/Asset/EntrypointLookup.php#L71 EntrypointLookup->getEntryFiles() method: assets are cached in  private $this->returnedFiles variable, so they can't be returned out of method.

I tried also to initialize the HttpKernel not in constructor but as new object in each https://github.com/franzwilding/last/blob/master/Service/SiteGenerator.php#L56 element of cycle without luck. I thought the problem was the client under the HttpKernel but maybe that's not.
